coralproject/talk
# [CORL-2685] username aria labels

## What does this PR do?
This addresses an accessibility concern by giving usernames in the stream aria labels of the form "User <username>".

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?
None.

## Does this PR introduce any new environment variables or feature flags? 
No.

## If any indexes were added, were they added to `INDEXES.md`?
n/a

## How do I test this PR?
Navigate to the stream, and inspect each instance of a username. Observe that usernames have aria labels of the form "User <username>".
 
 
## How do we deploy this PR?
No special considerations should necessary.
